### PR TITLE
Add test to create namespaces with different modes with single region

### PR DESCRIPTION
**Pull request by harish-24**
Add test to create namespaces with different modes with single region

Signed-off-by: Harish <harish@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by harish-24** [comment]
```# avocado run memory/ndctl.py:NdctlTest.test_multiple_ns_modes_region
...
 (1/1) memory/ndctl.py:NdctlTest.test_multiple_ns_modes_region: PASS (14.88 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 14.95 s```

File: memory/ndctl.py [hunk 1]
```diff
@@ -274,7 +274,9 @@ def test_namespace_modes(self):
         """
         failed_modes = []
         self.enable_region()
-        region = self.get_json(short_opt='-R')[0]
+        region = self.params.get('region', default=None)
```
Comment (narasimhan-v) [inline memory/ndctl.py]: Can't this be moved to setUp() ?

Comment (harish-24) [reply memory/ndctl.py]: yes, this particular line can be moved. but `self.get_json` in the line following cannot be used without enabling regions. So thought of keeping it together.

Comment (narasimhan-v) [reply memory/ndctl.py]: Ok. I see most of the tests are using this snippet. So the question.

Comment (PraveenPenguin) [reply memory/ndctl.py]: This  way we can assign variable in local scope if it not using any where else and avoid to use heap memory

**Pull request merged**